### DISCUSSION TITLE
Pagination in API [May 01]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#1999] [Discord Threads]
> 🙋‍♂️ **vishal.gandhi**

**Comment by o1lab** [comment]
> 👨‍🚀 **wingkwong**  
#### you can pass `offset` in query  
#  
> 👨‍🚀 **wingkwong**  
#### the value would be different for each page  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/969664927067406347)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
